modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
The red bar acts strangely a lot of the time though, like it'll load forward and then bounce back before reaching the end of the page. Happens a lot, like during a demo once and no one liked it. If the FileTable could get it's own query & spinner that'll be ideal. Hey it's not as bad as each Mutation Count row being it's own container.